File: d_1_adaptiveversion/src/main/java/com/sj/d_1_adaptiveversion/MainActivity.java
package com.sj.d_1_adaptiveversion;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.sj.d_1_adaptiveversion.version_10.UniqueActivity;
import com.sj.d_1_adaptiveversion.version_6.PermissionActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.version_6).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, PermissionActivity.class));
            }
        });
        findViewById(R.id.version_10).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, UniqueActivity.class));
            }
        });
    }
}

File: base_viewpager/src/main/java/com/sj/base_viewpager/MainActivity.kt
package com.sj.base_viewpager

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        infinite_tv.setOnClickListener(object :View.OnClickListener{
            override fun onClick(p0: View?) {
                startActivity(Intent(this@MainActivity,InfiniteVPActivity::class.java))
            }

        })
    }

}
